imgui-SFML.cpp

```diff
@@ -1,5 +1,6 @@
 #include "imgui-SFML.h"
 #include <imgui.h>
+#include <imgui_internal.h>
```

Comment:
Why is this needed?

Reply:
Due to necessary calls to ``ItemSize`` and ``ItemAdd``